examples/map_plots/custom_map_domain.py

```diff
@@ -1,31 +1,51 @@
+"""
+Create a custom map domain
+--------------------------
+
+EMCPy's map domain includes a feature
+to create a custom dictionary with specified
+lat and lon inputs to create a specific map
+domain. This is helpful for users who do want
```

Comment:
Suggestion on phrasing "This is helpful for users who want to look at a specific domain that is not included in EMCPy."